db/migrate/20240226165816_create_heads_of_terms.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateHeadsOfTerms < ActiveRecord::Migration[7.1]
+  def change
+    create_table :heads_of_terms do |t|
+      t.references :planning_application
+      t.timestamps
+    end
+
+    create_table :terms do |t|
+      t.string :title
+      t.text :text
```

Comment:
do we want to add a `null: false` since you're validating that presence is true in the model